### PR TITLE
patch ISUIElement to fit screen size, add util to mark stash locations on map

### PR DESCRIPTION
**Pull request by Poltergeistzx**

### CODE REVIEW

File: Contents/mods/!pz_debugTools/media/lua/client/Util_Functions_Community_client.lua [hunk 1]
```diff
@@ -0,0 +1,25 @@
+--local util = require "Util_Functions_Community_client.lua"
+local util = {}
+
+function util.markAllStasBuildingshOnMap(show,clear)
```
Comment (ChuckTheSheep) [inline Contents/mods/!pz_debugTools/media/lua/client/Util_Functions_Community_client.lua:4]: Stas**h** ;)

File: Contents/mods/!pz_debugTools/media/lua/client/vanillaDebugTweaks.lua [hunk 2]
```diff
@@ -21,4 +21,15 @@ function ISPlayerStatsUI:onOptionMouseDown(button, x, y)
     end
 
     ISPlayerStatsUI_onOptionMouseDown(self, button, x, y)
-end
\ No newline at end of file
+end
+
+--some debug panels expand beyond the limits of screen (Lootzed,Server Statistic(still too big for 720p),?)
```
Comment (ChuckTheSheep) [inline Contents/mods/!pz_debugTools/media/lua/client/vanillaDebugTweaks.lua:26]: Could have it be `screenHeight * 0.9`

Comment (Poltergeistzx) [reply Contents/mods/!pz_debugTools/media/lua/client/vanillaDebugTweaks.lua:26]: done

File: Contents/mods/!pz_debugTools/media/lua/client/Util_Functions_Community_client.lua (hunk 1 above)
Comment (Poltergeistzx) [reply Contents/mods/!pz_debugTools/media/lua/client/Util_Functions_Community_client.lua]: heh, that h is at wrong place. fixed typo

**Review by ChuckTheSheep — APPROVED** [review]

**Pull request merged**